lib/logger/test/logger/translator_test.exs

```diff
@@ -552,6 +570,34 @@ defmodule Logger.TranslatorTest do
     assert_receive {:event, {:string, ["Process " | _]}, _process_metadata}
   end
 
+  test "translates :gen_statem crashes when format_status/2 does not return a tuple" do
```

Comment:
Should we keep this test just in case to prevent similar regressions?
It feels like a bit artificial now that there is no code assuming a tuple in the first place.
No strong opinion.

Reply:
I would nuke it, yeah. :)

Reply:
Sorry, to be clear, we should have tests for both styles of `gen_statem`, but perhaps we should change its name.

Reply:
OK, like [578e043](https://github.com/elixir-lang/elixir/pull/13684/commits/578e0434c69f1107d20019b21a27ebc895b15787)?